### PR TITLE
Feat/services

### PR DESCRIPTION
**Pull request by wirayuda299**
Create service section

### CODE REVIEW

File: components/Services/Card.tsx
Comment (brandonetter) [inline components/Services/Card.tsx:1]: The design isn't trying to show you that one of the cards is blue and lower, it's an implied animation. The expectation is that a mouse over will cause the animation. Same for the skills icons, the implication isn't that there's one that is full color. The idea is that a hover will cause it to become colored.

**Review by brandonetter — CHANGES_REQUESTED** [review]
Small issue with the figma interpretation- but honestly this is looking BEAUTIFUL and spot-on with the design aside from that. Excellent work!

File: components/Services/Card.tsx
Comment (wirayuda299) [reply components/Services/Card.tsx:1]: ohh i see😁 

I will update my code, thanks for the feedback🙏

Comment (brandonetter) [reply components/Services/Card.tsx:1]: No problem at all! You're not the only one, no worries!

Comment (wirayuda299) [reply components/Services/Card.tsx:1]: Thanks, is that same as well with work experience section? when the right card is hover it will show the shadow?

Comment (brandonetter) [reply components/Services/Card.tsx:1]: Yes sir!

**Pull request merged**